Formula/c/container.rb

```diff
@@ -0,0 +1,47 @@
+class Container < Formula
+  desc "Create and run Linux containers using lightweight virtual machines"
+  homepage "https://apple.github.io/container/documentation/"
+  url "https://github.com/apple/container/archive/refs/tags/0.4.1.tar.gz"
+  sha256 "8a77933181577c4e459f75e392cdf8fadaede6b9917540278da3662fd712001b"
+  license "Apache-2.0"
+  head "https://github.com/apple/container.git", branch: "main"
+
+  depends_on xcode: ["26.0", :build]
+  depends_on arch: :arm64
+  depends_on macos: :sequoia
+  depends_on :macos
+
+  def install
+    if build.head?
+      ENV["GIT_COMMIT"] = Utils.git_head
+    else
+      ENV["RELEASE_VERSION"] = version
+    end
+
+    system "swift", "build", "--disable-sandbox", "--configuration", "release"
+
+    release_dir = buildpath/".build/release"
+    bin.install release_dir/"container"
+    bin.install release_dir/"container-apiserver"
+
+    (libexec/"plugins/container-core-images/bin").install release_dir/"container-core-images"
+    (libexec/"plugins/container-network-vmnet/bin").install release_dir/"container-network-vmnet"
+    (libexec/"plugins/container-runtime-linux/bin").install release_dir/"container-runtime-linux"
```

Comment:
These two binaries have relevant entitlements that are added by the upstream makefile, but even with them applied I haven't been able to get things working locally.

```ruby
system "/usr/bin/codesign", "-f", "-s", "-", "--prefix=com.apple.container.", "--entitlement", buildpath/"signing/container-network-vmnet.entitlements", release_dir/"container-network-vmnet"
    system "/usr/bin/codesign", "-f", "-s", "-", "--prefix=com.apple.container.", "--entitlement", buildpath/"signing/container-runtime-linux.entitlements", release_dir/"container-runtime-linux"
```